homeassistant/components/intent/timers.py

```diff
@@ -131,6 +147,7 @@ def add_time(self, seconds: int) -> None:
         Seconds may be negative to remove time instead.
         """
         self.seconds = max(0, self.seconds_left + seconds)
+        self._created_seconds = max(self._created_seconds, self.seconds)
```

Comment:
Why not always do this? Wouldn’t that be more expected behavior

Reply:
If you have a 10 minute timer that has 5 minutes left and you add 1 minute, would you expect the progress bar to go back to 100% or to go to 60%?

Reply:
I would expect it to go from 300/600 to 360/660, which is 54.5%

Reply:
That seems very odd to me, and gets even weirder if you repeat the action. At 5 minutes left, you add a minute to the timer. Now the bar is at 54.5% (360/660). Wait another minute, then add a minute. Now the bar is at 50%. Another round, and the bar is at 46.1%.

Reply:
As discussed in Discord, Paulus agreed with Mike proposal.
A timer's total number of seconds will only grow in size if you add more time than its original duration.

Reply:
Understood on discord it was agree this is the right path.